src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime()
+        if(!image) {
+            return
+        }
+        try {
+            retry(3) {
+                def dd = e.git.url
+                echo "YZAINEE: ${dd}"
+                def response = bayesianAnalysis url: 'https://bayesian-link', gitUrl: e.git.url, ecosystem: image
+                if (response.success) {
+                  Utils.addAnnotationToBuild(this, 'fabric8.io/bayesian.analysisUrl', response.getAnalysisUrl())
```

Comment:
have you tried if `response.analysisUrl` works? in groovy `obj.getFoobar` can be called as `obj.foobar` which is why I am asking.

Reply:
yes it works

Reply:
@yzainee as @sthaha mentioned, did you try `response.analysisUrl` instedof `response.getAnalysisUrl()`? If yes and `response.analysisUrl` works then could you please update the patch?

Reply:
@yzainee if  `response.analysisUrl` works could you please change to that?

Reply:
@sthaha I have already tried it.. it doesnt work.. Had updated the PR comments accordingly. PTAL